blog/2024/05-05-jellyfin-release-10.9.0/index.mdx

```diff
@@ -0,0 +1,133 @@
+
+### Transcoding & FFmpeg
+
+* The `--ffmpeg` command-line flag is now the primary method of setting FFmpeg paths, and configuration of the FFmpeg paths via `encoding.xml` is now deprecated. All official packaging will revert to defaults unless explicitly set before upgrade.
+
+* Several audio improvements: support for Opus, FLAC, and ALAC codecs in the HLS engine; improved support and handling for transcoding DTS and TrueHD audio streams; new codec enforcement; better bitrate calculations; new audio normalization features; and stereo downmixing algorithm selection.
+
+* FFmpeg segments can now be automatically deleted after the client requests them, significantly reducing the space requirements of the transcoding temporary directory. This optional feature is disabled by default and can be enabled in the transcoding settings.
+
+* Support for direct stream playback of DVD and BluRay data folders, preferred over ISOs.
+
+* Support for AV1 hardware and software encoding.
+
+* Support for RK3588 hardware acceleration via MPP.
+
+* Support for Vulkan-based tonemapping with AMD VA-API.
+
+* Support for complete hardware acceleration on Apple systems (both Apple Silicon and Intel; previously was partial).
+
+* Support for Dolby Vision playback remuxing.
+
+### Scanning, Library & Playlist Management
+
+* Support for scanning and displaying lyrics.
+
+* Season names are now parsed from NFO files.
+
+* The MusicBrainz interface has been improved to better handle music libraries.
+
+* Support for hearing-impaired subtitles (sdh/hi./cc).
+
+* Support for playlist ACLs with playlist sharing between users and enforcement of playlist ownership. Existing playlists will become owner-only, and playlist permissions can be set when made global/shared. Playlists from library files will be global.
+
+* Invalid items will be automatically removed from playlists.
+
+### Clients
+
+* Major fixes for Chromecast, including better support for a self-hosted cast receiver, better local network detection, and connectivity improvements. **We need Chromecast client developers badly - if you are interested please reach out.**
+
+## The Next Version
+
+With our CI improvements outlined previously, we're quite confident that this will be our last "very long" release cycle. Our plan is for the next major version (10.10.0) to be released at most 6 months from now, some time in October. We hope this increased cadence will help alleviate the problems with large releases such as a very long time-to-stable for new features, translations, etc. and help lower the number of major bugs at each major release, streamlining the upgrade process. But this needs everyone's help. Back in October 2023, we made a call for developers, and we've gotten a lot of interest, but this is not a one-and-done event. We need contibutions now more than ever, especially around the web frontend to help implement our planned design changes. If this interests you, please reach out and we can help get you set up.
```

Comment:
```suggestion
With our continuous integrations improvements outlined previously, we're quite confident that this will be our last "very long" release cycle. Our plan is for the next major version (10.10.0) to be released at most 6 months from now, some time in October. We hope this increased cadence will help alleviate the problems with large releases such as a very long time-to-stable for new features, translations, etc. and help lower the number of major bugs at each major release, streamlining the upgrade process. But this needs everyone's help. Back in October 2023, we made a call for developers, and we've gotten a lot of interest, but this is not a one-and-done event. We need contributions now more than ever, especially around the web frontend to help implement our planned design changes. If this interests you, please reach out and we can help get you set up.
```